aspnetcore/release-notes/aspnetcore-9.0.md

```diff
@@ -47,12 +47,8 @@ This section describes new features for OpenAPI
 
 [!INCLUDE[](~/release-notes/aspnetcore-9/includes/openAPI_completion.md)]
```

Comment:
We can remove the completion section too since it requires some additional bug fixes.